src/GovUk.Education.ExploreEducationStatistics.Content.Model/Repository/ReleaseVersionRepository.cs

```diff
@@ -172,6 +172,23 @@ public async Task<List<ReleaseVersion>> ListLatestReleaseVersions(CancellationTo
                releaseVersion.Version;
     }
 
+    public async Task<ReleaseVersion?> GetLatestReleaseVersionForParent(
+        Guid releaseId,
+        bool publishedOnly = false,
+        CancellationToken cancellationToken = default)
+    {
+        var maxVersion = await GetMaxVersionNumber(
+            releaseId,
+            publishedOnly: publishedOnly,
+            cancellationToken: cancellationToken);
+
+        return await _contentDbContext.ReleaseVersions
+            .Where(rv =>
+                rv.ReleaseId == releaseId
+                && rv.Version == maxVersion)
+            .FirstOrDefaultAsync(cancellationToken);
+    }
```

Comment:
We have unit tests for all of the latest release version methods in this repository and I noticed this addition hasn't added any tests.

I think I would add a unit test in for consistency but also because these methods tend to be quite heavily relied upon.

In another comment I've suggested you could move this to `ReleaseVersionPredicates` but just noticed that class itself doesn't have it's own tests. I guess if they end up being tested indirectly via another method then it's ok to omit the tests.